pkg/services/business_case.go

```diff
@@ -210,21 +223,9 @@ func NewUpdateBusinessCase(
 				Operation: apperrors.QuerySave,
 			}
 		}
-		// Right now, add some validations for sending an email here
-		// Similar to system intake,
-		// these should be covered by validations above
-		if businessCase.Status == models.BusinessCaseStatusSUBMITTED &&
-			existingBusinessCase.Status == models.BusinessCaseStatusDRAFT {
-			if !businessCase.Requester.Valid {
-				validationError := apperrors.NewValidationError(
-					errors.New("failed to validate for email"),
-					businessCase,
-					businessCase.ID.String(),
-				)
-				validationError.WithValidation("Requester", "is required")
-				logger.Error("Failed to validate", zap.Error(&validationError))
-				return businessCase, &validationError
-			}
+
+		// At this point, if everything has gone well, email the GRT
+		if businessCase.Status == models.BusinessCaseStatusSUBMITTED {
```

Comment:
I think this needs to be fixed both here and system intake (I may not have added it to system, because it's not editable after). Since a business case will eventually be edited after submission, we need to only send this email at the state transition (ie. include `existing == DRAFT`